README.md

```diff
@@ -29,10 +29,10 @@ MailHog is an email testing tool for developers:
 brew update && brew install mailhog
 ```
 
-Pro tip: Use brew services to start Mailhog and have it automatically started when your computer restarts: `brew services start mailhog`.
-
 Then, start MailHog by running `mailhog` in the command line.
 
+Pro tip: Use brew services to start Mailhog and have it automatically started when your computer restarts: `brew services start mailhog`.
```

Comment:
question: Is brew not macOS only?

If so, I guess we could add a notice or similar commands for Linux (SystemD: `systemctl --user edit mailhog.service --full --force` *and fill the config file*) and Windows (`sc.exe create mailhog binpath= "c:\Program Files\mailhog" start= auto`).

Reply:
Brew is only for macOS. I feel like I remember hearing about someone using it on another platform, but that's not the intended use. Happy to add those 2 comments that you put, but I'm not sure if I'm quite following where you'd want them of what you mean by fill in the config file. Mind elaborating a bit and I'll make the changes? 🙂

Reply:
@C-Duv Wanted to bump this. 👀